lxd/api_internal.go

```diff
@@ -129,6 +129,16 @@ func internalWaitReady(d *Daemon, r *http.Request) response.Response {
 func internalShutdown(d *Daemon, r *http.Request) response.Response {
 	d.shutdownChan <- struct{}{}
 
+	var req map[string]interface{}
```

Comment:
Can we make that ?force=true instead? That'd be easier to use through `lxc query` and we have precedent for doing something like that with the `callhook` endpoint.